### PR TITLE
#7595 Caches of different events don't show favorite points

### LINKED ISSUE #7595: Do not show favorite count for caches that cannot receive favorite points
Event caches cannot receive favorite points (at least on geocaching.com, not sure about opencaching) but we always show a line "Favorite 0×" in the cache details. It would be nice if we remove the line for such caches.

### PR DESCRIPTION
**Pull request by emilvidmark**
Fixes #7595

** Description of change: **
Caches that isn't able to have favorite points no longer shows the favorite points on cache details and cache popup. (As far as I know it's only caches of different event types.).

** Screenshot (if applicable): **
Caches with favorite points:

Screenshot_20191006-212827_cgeo
Screenshot_20191006-212836_cgeo

Caches that can't have favorite points:

Screenshot_20191006-212907_cgeo
Screenshot_20191006-212915_cgeo

### CODE REVIEW

File: main/src/cgeo/geocaching/AbstractDialogFragment.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import cgeo.geocaching.activity.AbstractActivity;
 import cgeo.geocaching.activity.ActivityMixin;
+import cgeo.geocaching.enumerations.CacheType;
```
Comment (UniQP) [inline main/src/cgeo/geocaching/AbstractDialogFragment.java]: The import is superfluous.

**Comment by emilvidmark** [comment]
@UniQP Thank you, I've fixed the changes you commented.

File: main/src/cgeo/geocaching/AbstractDialogFragment.java [hunk 2]
```diff
@@ -220,7 +220,8 @@ protected final void addCacheDetails() {
             final int findsCount = cache.getFindsCount();
             if (findsCount > 0) {
                 details.add(R.string.cache_favorite, res.getString(R.string.favorite_count_percent, favCount, (float) (favCount * 100) / findsCount));
-            } else {
+            }
+            else if (!cache.isEventCache()) {
```
Comment (bekuno) [inline main/src/cgeo/geocaching/AbstractDialogFragment.java]: please place it on one row
~~~
            } else if (!cache.isEventCache()) {
~~~

Comment (emilvidmark) [reply main/src/cgeo/geocaching/AbstractDialogFragment.java]: My fault, it's fixed now.

**Pull request merged**